recipes/nuklear/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "4.02.1":
+    url: "https://github.com/Immediate-Mode-UI/Nuklear/archive/9f0bca461b028c1f8b638beeba1859045ebe1ac3.zip"
```

Comment:
How do you know that this version matches with this commit?

Reply:
https://github.com/Immediate-Mode-UI/Nuklear/commit/9f0bca461b028c1f8b638beeba1859045ebe1ac3#diff-54af4b6cab9b8bb5a911bbc6b9f4a21aR29094

Reply:
Thank you! I wish people tag their own releases... 😿